SpatialGDK/Documentation/content/cloud-deployment-workflow.md

```diff
@@ -26,6 +28,8 @@ Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat <YourProject> Wi
 
 Replacing `<YourProject>` with the name of your Unreal project.
 
+For more information on the available options when using `BuildWorker.bat`, please see the [Helper scripts reference]({{urlRoot}}/content/helper-scripts).
```

Comment:
Just wanna doublecheck- are we planning on publishing these docs on github as well as on the website? Having trouble following this link as it stands (manually followed it and page looks correct - but want to doublecheck in case it should also follow through on github)

Reply:
no our docs are not meant to be readable on github - just docs.improbable.io - and we signpost this extensively

Reply:
That's fine with me :) in this case content all LGTM, thanks a lot for these changes!

Reply:
cool thanks :)